packages/editor-ui/package.json

```diff
@@ -14,33 +14,63 @@
     "url": "git+https://github.com/n8n-io/n8n.git"
   },
   "scripts": {
-    "build": "cross-env VUE_APP_PUBLIC_PATH=\"/%BASE_PATH%/\" vue-cli-service build",
+    "build": "cross-env VUE_APP_PUBLIC_PATH=\"/%BASE_PATH%/\" vite build",
     "dev": "npm run serve",
-    "format": "cd ../.. && node_modules/prettier/bin-prettier.js packages/editor-ui/**/**.ts --write",
     "lint": "tslint -p tsconfig.json -c tslint.json && eslint .",
     "lintfix": "tslint --fix -p tsconfig.json -c tslint.json && eslint . --fix",
-    "serve": "cross-env VUE_APP_URL_BASE_API=http://localhost:5678/ vue-cli-service serve",
-    "test:e2e": "vue-cli-service test:e2e",
-    "test:unit": "vue-cli-service test:unit"
+    "format": "cd ../.. && node_modules/prettier/bin-prettier.js packages/editor-ui/**/**.ts --write",
+    "serve": "cross-env VUE_APP_URL_BASE_API=http://localhost:5678/ vite --host 0.0.0.0 --port 8080 dev",
+    "test": "# vitest run",
```

Comment:
`#`?

Reply:
> > Found this error on 5678 when switching locale:
> > ```
> > Uncaught (in promise) Error: Unknown variable dynamic import: ./locales/de.json
> >     __variableDynamicImportRuntime0__ http://localhost:5678/assets/index.754241ce.js:16669
> >     __variableDynamicImportRuntime0__ http://localhost:5678/assets/index.754241ce.js:16669
> >     loadLanguage http://localhost:5678/assets/index.754241ce.js:16669
> >     defaultLocale http://localhost:5678/assets/index.754241ce.js:16669
> >     invokeWithErrorHandling http://localhost:5678/assets/index.754241ce.js:5
> >     run http://localhost:5678/assets/index.754241ce.js:5
> >     flushSchedulerQueue http://localhost:5678/assets/index.754241ce.js:5
> >     nextTick http://localhost:5678/assets/index.754241ce.js:5
> >     flushCallbacks http://localhost:5678/assets/index.754241ce.js:5
> >     promise callback*timerFunc http://localhost:5678/assets/index.754241ce.js:5
> >     nextTick http://localhost:5678/assets/index.754241ce.js:5
> >     queueWatcher http://localhost:5678/assets/index.754241ce.js:5
> >     update http://localhost:5678/assets/index.754241ce.js:5
> >     notify http://localhost:5678/assets/index.754241ce.js:5
> >     set http://localhost:5678/assets/index.754241ce.js:5
> >     set$1 http://localhost:5678/assets/index.754241ce.js:5
> >     setDefaultLocale http://localhost:5678/assets/index.754241ce.js:16874
> >     registerMutation http://localhost:5678/assets/index.754241ce.js:609
> >     commit http://localhost:5678/assets/index.754241ce.js:609
> >     commit http://localhost:5678/assets/index.754241ce.js:609
> >     _withCommit http://localhost:5678/assets/index.754241ce.js:609
> >     commit http://localhost:5678/assets/index.754241ce.js:609
> >     commit http://localhost:5678/assets/index.754241ce.js:609
> >     commit http://localhost:5678/assets/index.754241ce.js:609
> >     getSettings http://localhost:5678/assets/index.754241ce.js:16874
> >     registerAction http://localhost:5678/assets/index.754241ce.js:609
> >     dispatch http://localhost:5678/assets/index.754241ce.js:609
> >     dispatch http://localhost:5678/assets/index.754241ce.js:609
> >     initSettings http://localhost:5678/assets/index.754241ce.js:16669
> >     initialize http://localhost:5678/assets/index.754241ce.js:16669
> >     mounted http://localhost:5678/assets/index.754241ce.js:16669
> >     invokeWithErrorHandling http://localhost:5678/assets/index.754241ce.js:5
> >     callHook http://localhost:5678/assets/index.754241ce.js:5
> >     insert http://localhost:5678/assets/index.754241ce.js:5
> >     z http://localhost:5678/assets/index.754241ce.js:5
> >     createPatchFunction http://localhost:5678/assets/index.754241ce.js:5
> >     _update http://localhost:5678/assets/index.754241ce.js:5
> >     u http://localhost:5678/assets/index.754241ce.js:5
> >     get http://localhost:5678/assets/index.754241ce.js:5
> >     Watcher http://localhost:5678/assets/index.754241ce.js:5
> >     mountComponent http://localhost:5678/assets/index.754241ce.js:5
> >     $mount http://localhost:5678/assets/index.754241ce.js:5
> >     <anonymous> http://localhost:5678/assets/index.754241ce.js:16874
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > To reproduce, create `packages/editor-ui/src/plugins/i18n/locales/de.json`, edit all values to make it visually clear it is a different locale, `export N8N_DEFAULT_LOCALE=de` on 5678 and start. Editing the `de.json` triggers hot reload on 8080, but its translated strings are not used. [Docs link](https://github.com/n8n-io/n8n/blob/master/packages/editor-ui/src/plugins/i18n/docs/README.md#building-translations).
> 
> Good catch, working on fixing it!

Looks like we both contributed a bit to this 😅. Fixed!